descriptors/ModuleDescriptor-template.json

```diff
@@ -1179,6 +1179,27 @@
         }
       ]
     },
+    {
+      "id": "orders.send-claims",
```

Comment:
what do you think about name pices.claims or pieces.send-claims?
because we sending claims for pieces now, not for orders